src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -136,10 +136,10 @@ public sealed class Settings
 #pragma warning restore MSML_NoInstanceInitializers // No initializers on instance fields or properties
         }
```

Comment:
This whole thing requires proper cleaning/documentation. Do you want to do it in this PR, or you prefer leave it for later cleaning?

Reply:
The current PR is to fix the public API

We are doing some cleaning/doc fixes on an opportunistic basis. But we should have a separate issue / PRs  for "proper" cleaning and documentation

---
In reply to: [254013725](https://github.com/dotnet/machinelearning/pull/2394#discussion_r254013725) [](ancestors = 254013725)